handlers/buy.go

```diff
@@ -100,21 +100,29 @@ func Balance(s interaction.Session, m *interaction.MessageCreate, db *sqlx.DB) {
 	if len(args) == 1 {
 		author := UserGet(m.Author, db)
 		if author.CurMoney < 0 {
-			author.CurMoney = (author.CurMoney * -1)
+			author.CurMoney = author.CurMoney * -1
 			MoneySet(&author, author.CurMoney, db)
 		}
 		message := author.Username + "\r"
 		message = message + "Prestige Level " + strconv.Itoa(author.PrestigeLevel) + "\r"
 		message = message + "total balance is: " + humanize.Comma(int64(author.CurMoney))
 		_, production, _ := ProductionSum(m.Author, db)
 		production = PrestigeBonus(production, &author)
-		if (production) < 10 {
-			message = message + "\ntotal memes per minute: " + Ftoa((float64(production) / 10))
-		} else {
-			message = message + "\ntotal memes per minute: " + humanize.Comma(int64((float64(production) / 10)))
-		}
+
+		// The user will probably want to know their meme balance down to the tenth regardless of whether they have less
+		// than one meme per second or not
+
+		//if production < 10 {
```

Comment:
dont comment it out, delete the old code

Reply:
Fixed